.github/workflows/deploy-to-control-plane.yml

```
@@ -0,0 +1,404 @@
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      deployments: write
+      pull-requests: write
+      issues: write
+
+    steps:
+      - name: Get PR HEAD Ref
+        if: github.event_name == 'issue_comment'
+        id: getRef
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:22: Double quote to prevent globbing and word splitting

(shellcheck)